.github/pull_request_template.md

```diff
@@ -0,0 +1,39 @@
+Please use the following template to submit a pull request, filling
+in at least the "Text for release notes" and/or "Further details".
+Thank You!
+
+# Description
+
+## Text for release notes 
+
+If this pull request should be mentioned in the release notes, 
+please provide its short description matching the style of the GAP
+[Changes manual](https://www.gap-system.org/Manuals/doc/changes/chap0.html).
+
+## Further details
+
+If necessary, please provide further details here.
+
+# Checklist for pull request reviewers
```

Comment:
I am not sure whether putting this into the PR template is a good idea I mean this honestly (i.e., not a British way to say that I hate it ;-) -- maybe it is a great idea, I just am unsure about it. Thing is, this is a checklist for reviewers, but we are asking submitters to put it in, which seems wrong to me. I guess it'll work if most PR submitters will just leave it alone; but they can easily delete or even modify it... Hm

Reply:
Yes, I also thought about some alternatives, not feeling entirely satisfied with this. 

One alternative that came to my mind is that everyone who is reviewing PRs is advised to use https://github.com/settings/replies to create a saved reply with PR review template. Then they will be able to post it in comments under PR, at any place - just as a comment, or when submitting review. 

Pluses:
- Plus: it does not clutter the PR description
- Plus: each reviewer can fill in their template independently
- Plus: PR submitter is not distracted by the checklist (which has things like labels which they might not be able to assign, or refer to tests which results they can not see yet)
- Does not require merging this PR to start practical use of the template

Minuses:
- people will have to synchronise their copy of the template in case of future updates
- reviewers will have to remember to use this template
- the report may be less visible after all comments under the initial PR description 
- its less convenient to fill in the checklist in a comment than tick boxes in another window
- PR submitters will miss the guidance that the checklist provides

Perhaps we should try to start to use it and see how it will go...

Reply:
Sure, fine by me